scripts/ci_install_dependency.sh

```diff
@@ -4,5 +4,5 @@ Install the dependency in CI.
 
 pip install --upgrade pip
 pip install -e "python[all]"
-pip install transformers==4.45.2 sentence_transformers
+pip install transformers==4.45.2 sentence_transformers accelerate peft
```

Comment:
why do we need these?

Reply:
some unit tests such as lora needs this.